pkgs/development/libraries/maxflow/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maxflow";
+  version = "3.0.5";
+
+  src = fetchFromGitHub {
+    owner = "gerddie";
+    repo = pname;
+    rev = version;
+    hash = "sha256-a84SxGMnfBEaoMEeeIFffTOtErSN5yzZBrAUDjkalGY=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with lib; {
+    description = "Software for computing mincut/maxflow in a graph";
+    homepage = "https://github.com/gerddie/maxflow";
+    license = with licenses; [ gpl2Only gpl3Only ];
```

Comment:
Where did you get the license? Looks GPL3+ to me: https://github.com/gerddie/maxflow/search?q=license

Reply:
My bad, I'll change this. I was going off of the license description in the README.